packages/palette/src/Theme.tsx

```diff
@@ -95,6 +101,11 @@ export const themeProps = {
 
   fontFamily,
 
+  fonts: TEXT_FONTS,
+  fontSizes: TEXT_FONT_SIZES,
+  letterSpacings: TEXT_LETTER_SPACING,
+  lineHeights: TEXT_LINE_HEIGHTS,
```

Comment:
These follow the styled-system theme specification. There's a type they export but we can't use it at the moment.